app/utility/transforms/Validator.js

```diff
@@ -0,0 +1,9 @@
+import OpenBarbellConfig from 'app/configs/OpenBarbellConfig.json';
+
+export const validator = () => {
+    if (this.lastRefreshDate < OpenBarbellConfig.obtainTokenTimer) {
```

Comment:
If it's just an if statement, can return (this.lastRefreshDate < OpenBarbellConfig.obtainTokenTimer)